client/keys.go

```diff
@@ -184,6 +184,10 @@ type SetOptions struct {
 	// a TTL of 0.
 	TTL time.Duration
 
+	// Refresh specified whether or not a key with a TTL will
+	// fire a watch return when a SET is performed
+	Refresh bool
```

Comment:
let's make this more clear. If refresh is set to true, value should not be set, and when the SET succeeds it will not trigger any watchers.